### ISSUE TITLE
Misuse "[EA] This condition execution costs less than the previous one"

### ISSUE THREAD

**Issue by deMoneIrk** [kalessil/phpinspectionsea#1645]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Php Inspections (EA Ultimate) 3.0.10 |
| Language level | PHP 7.4                        |

Current behaviour
--
Code:
```php
public function setCreatedBy() : void
    {
        if ($this->getCreatedByColumn() && null === $this->{$this->getCreatedByColumn()})
        {
            $this->{$this->getCreatedByColumn()} = Auth::id();
        }
    }
```

I have a notice about `null === $this->{$this->getCreatedByColumn()}`: [EA] This condition execution costs less than the previous one, but it can't be right just because this condition little bit more difficult, that just `$this->getCreatedByColumn()`

Expected behaviour
--
Notice shouldn't be shown

Environment details
---
PhpStorm 2020.2.2
Build #PS-202.7319.77, built on September 23, 2020
Licensed to Tech-Director.ru / EVGENIY NEVEROV
Subscription is active until October 22, 2020
Runtime version: 11.0.8+10-b944.31 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 10.16
GC: ParNew, ConcurrentMarkSweep
Memory: 4012M
Cores: 12
Registry: run.processes.with.pty=TRUE
Non-Bundled Plugins: com.alayouni.ansiHighlight, com.intellij.ideolog, mobi.hsz.idea.gitignore, com.dmarcotte.handlebars, com.jetbrains.plugins.meteor, de.espend.idea.laravel, com.kalessil.phpStorm.phpInspectionsUltimate, pro.opcode.bitrix, de.espend.idea.php.toolbox, de.espend.idea.php.annotation, fr.adrienbrault.idea.symfony2plugin, ru.adelf.idea.dotenv

**Comment by ea-inspections-team** [comment]
Thanks for reporting @deMoneIrk! Fixed!
